src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -17,6 +17,10 @@ param exportContainerName string
 @description('Required. The name of the container where normalized data is ingested.')
 param ingestionContainerName string
 
+@description('Optional. The name of the container where normalized data is ingested.')
+param configContainerName string = 'config'
```

Comment:
nit: Do you want to out a default? Since this is internal, it's fine for us to make it required to protect us from future changes if we rename something.

Reply:
Good point.